package.json

```diff
@@ -24,21 +25,21 @@
     "react": "^16.8.6",
     "react-bootstrap": "1.0.0-beta.16",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.2.0",
     "react-icons": "^3.8.0",
     "react-router-dom": "^5.0.1",
+    "react-scripts": "3.2.0",
     "reflect-metadata": "^0.1.13",
-    "sequelize": "^5.21.2",
+    "sequelize": "^5.21.3",
     "sequelize-cli": "^5.5.1",
-    "sqlite3": "^4.1.0",
+    "sqlite3": "^4.1.1",
     "wait-on": "^3.3.0"
   },
   "peerDependencies": {
     "jquery": "^3.4.0",
     "popper.js": "^1.14.3"
   },
   "scripts": {
-    "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000 && electron .\"",
+    "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000 && electron --disable-gpu .\"",
```

Comment:
This will disable hardware acceleration for all users. Feel free to add this flag to a new script, calling it something like `"start-no-gpu"`.